.github/workflows/reanimated-compatibility-check-nightly.yml

```
@@ -61,7 +61,7 @@ jobs:
       - name: Install React Native CLI
         run: yarn add @react-native-community/cli
       - name: Create app
-        run: yarn rnc-cli init ${{env.APP_NAME}} --version ${{ matrix.react-native.version }} --skip-install --pm yarn --install-pods 0 --skip-git-init
+        run: npx @react-native-community/cli init ${{env.APP_NAME}} --version ${{ matrix.react-native.version }}  --skip-install --pm yarn --pm npm --install-pods 0 --skip-git-init
```

```suggestion
        run: npx @react-native-community/cli init ${{env.APP_NAME}} --version ${{ matrix.react-native.version }}  --skip-install --pm yarn --install-pods 0 --skip-git-init
```

Is this `--pm npm` needed? I think it overrides the previously specified package manager (i.e. `--pm yarn`) and I don't think this is a correct approach. I feel that we should stick to the `yarn` package manager.